packages/http-client-csharp/generator/TestProjects/Local/Sample-TypeSpec/src/Generated/Models/SampleTypeSpecContext.cs

```diff
@@ -6,13 +6,25 @@
 #nullable disable
 
 using System.ClientModel.Primitives;
+using SampleTypeSpec.Models.Custom;
 
 namespace SampleTypeSpec
 {
     /// <summary>
     /// Context class which will be filled in by the System.ClientModel.SourceGeneration.
     /// For more information <see href='https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/System.ClientModel/src/docs/ModelReaderWriterContext.md' />
     /// </summary>
+    [ModelReaderWriterBuildable(typeof(Thing))]
```

Comment:
Add test cases that validate these attributes exist outside of the generated sample as unit tests.

The test should include at least one case where a Model has a property of another model from a dependency.